qiskit_braket_provider/providers/braket_backend.py

```diff
@@ -327,8 +329,21 @@ def run(self, run_input, **options):
             del options["meas_level"]
 
         verbatim = options.pop("verbatim", False)
-        gateset = self._get_gateset() if not verbatim else None
-        braket_circuits = [to_braket(circ, gateset, verbatim) for circ in circuits]
+        native = options.pop("native", False)
+        gateset = self._get_gateset()
```

Comment:
Might be worth adding an option to `_get_gateset` to get the native gateset, so you don't need to get the non-native gateset each time

Reply:
I think I like how that looks, thanks